app/views/declaration/AgentDeclarationView.scala.html

```diff
@@ -18,83 +18,100 @@
+    }
+}
+
 @main_template(
-    title = s"${errorPrefix(form)}${messages("declaration.title")}"
+    title = s"${errorPrefix(form)}${messages("declaration.title")}",
+    showBackLink = true
 ) {
 
     @formHelper(action = AgentDeclarationController.onSubmit(), 'autoComplete -> "off") {
 
-        @components.back_link()
+        @errorSummary(form.errors)
 
-        @components.error_summary(form.errors)
+        <fieldset class="govuk-fieldset">
 
-        <fieldset>
-
-            <legend>
-                @components.heading("declaration.heading")
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                @headingComp("declaration.heading")
             </legend>
 
-            @components.input_text(
+            @inputText(
                 field = form("firstName"),
+                inputClass = Some("govuk-!-width-one-half"),
                 label = messages("declaration.firstName"),
-                autoComplete = Some("given-name")
+                autocomplete = Some("given-name")
             )
 
-            @components.input_text(
+            @inputText(
                 field = form("middleName"),
+                inputClass = Some("govuk-!-width-one-half"),
                 label = messages("declaration.middleName")
             )
 
-            @components.input_text(
+            @inputText(
                 field = form("lastName"),
+                inputClass = Some("govuk-!-width-one-half"),
                 label = messages("declaration.lastName"),
-                autoComplete = Some("family-name")
+                autocomplete = Some("family-name")
             )
 
-            @components.input_text(
+            @inputText(
                 field = form("agencyName"),
+                inputClass = Some("govuk-!-width-one-half"),
                 label = messages("declaration.agencyName")
             )
 
-            @components.input_text(
+            @inputText(
                 field = form("telephoneNumber"),
                 label = messages("declaration.telephoneNumber"),
-                inputClass = Some("form-control-3-4"),
-                autoComplete = Some("tel")
+                inputClass = Some("govuk-!-width-one-half"),
+                autocomplete = Some("tel"),
+                inputType = "tel"
             )
 
-            @components.input_text(
+            @inputText(
                 field = form("crn"),
                 label = messages("declaration.crn"),
-                inputClass = Some("form-control-3-4")
+                inputClass = Some("govuk-!-width-three-quarters"),
             )
 
         </fieldset>
 
         @if(appConfig.declarationEmailEnabled) {
             <p>@messages("declaration.paragraph1")</p>
             <p>@messages("declaration.paragraph2")</p>
-            @components.input_text(
+            @inputText(
                 field = form("email"),
                 inputClass = Some("declaration-email"),
```

Comment:
Is class "declaration-email" defined ?

Reply:
No it's probably old... I can look at what it does but I was going to ask why it's outside of the `<fieldset>` in the first place.

The appConfig for declarationEmailEnabled is set to `false` locally - but is it ever on? (This topic came up in UR since it was in the prototype)

Reply:
Question has been answered, I've removed the TODO and replaced this class